README.md

```diff
@@ -63,6 +64,28 @@ err := c.Provide(&Type1{Name: "I am an thing"})
 ```
 
 
+### Provide an Maps, slices or arrays
+
+Dig is now able to support maps, slices and arrays as objects to
+resolve, or provided as a dependency to the constructor.
+
+```go
+c := dig.New()
+var (
+	typemap = map[string]int{}
+	typeslice = []int{1, 2, 3}
+	typearray = [2]string{"one", "two"}
+)
+err := c.ProvideAll(typemap, typeslice, typearray)
+
+var resolveslice []int
+err := c.Resolve(&resolveslice)
+
+c.Invoke(func(map map[string]int, slice []int, array [2]string) {
```

Comment:
hmmm... this is actually kind of scary...

Am I reading this right - there can only be 1 dep with a `[]int` here?